http.js

```diff
@@ -1,8 +1,8 @@
 const OError = require('./index')
 
 class HttpError extends OError {
-  constructor(options) {
-    super(options)
+  constructor({ message, info, ...options }) {
```

Comment:
I went for the minimal changes on this one, because I was not sure what to do with the HttpErrors. Following discussion internally (https://github.com/overleaf/issues/issues/2393), we decided to remove HttpErrors, so I'll do that now.